.github/workflows/wf-device-camera.yml

```diff
@@ -6,13 +6,16 @@ jobs:
   sync-repo:
     runs-on: ubuntu-latest
     steps:      
-      - name: Build and version device-camera
-        uses: canonical/edgex-sync-and-create-launchpad-branch-action@v1.4.0
-        with:
-          edgex-repo: "device-camera-go"
-          canonical-repo: "edgex-device-camera-go"
-          patch-file: "0001-snapcraft-version-patch.diff"
-          ssh-private-key: ${{ secrets.SSH_CANONICAL_EDGEX_DEVICE_CAMERA }}
+    - name: Build and version device-camera
```

Comment:
Should version still be included in the name?

Reply:
In this case, "version" refers to the process of creating the VERSION file, which is still done there - it's just the patching of the versioning logic which has been removed.